charts/routes/templates/certs.yaml

```diff
@@ -1,12 +1,14 @@
 {{- $hosts := (concat (get $.Values.global.hosts "mlp") (get $.Values.global.hosts "mlpdocs") (get $.Values.global.hosts "mlflow") )}}
-{{- if and .Values.https.enabled }}
+{{- if .Values.https.enabled }}
+{{- if .Values.https.certificateIssuer.create }}
 ---
```

Comment:
This will create an extra `---` in the start of the file if https.enabled is true and certIssuer.create is false.

if I remember right, This will not execute right if there is no yaml before the `---` separator. Can you try it in your local cluster to confirm,

Reply:
just tried - it seems to be okay